frame/election-provider-multi-phase/src/unsigned.rs

```diff
@@ -17,22 +17,44 @@
 
 //! The unsigned phase implementation.
 
-use crate::*;
-use frame_support::dispatch::DispatchResult;
+use crate::{
+	helpers,
+	Call,
+	CompactAccuracyOf,
+	CompactOf,
+	CompactVoterIndexOf,
+	Config,
+	ElectionCompute,
+	Error,
+	FeasibilityError,
+	Pallet,
+	RawSolution,
+	ReadySolution,
+	RoundSnapshot,
+	SolutionOrSnapshotSize,
+	Weight,
+	WeightInfo,
+};
+use codec::Decode;
+use frame_support::{dispatch::DispatchResult, ensure, traits::Get};
 use frame_system::offchain::SubmitTransaction;
+use sp_arithmetic::Perbill;
 use sp_npos_elections::{
-	seq_phragmen, CompactSolution, ElectionResult, assignment_ratio_to_staked_normalized,
+	assignment_ratio_to_staked_normalized,
 	assignment_staked_to_ratio_normalized,
+	is_score_better,
+	seq_phragmen,
+	CompactSolution,
+	ElectionResult,
 };
 use sp_runtime::{offchain::storage::StorageValueRef, traits::TrailingZeroInput};
-use sp_std::cmp::Ordering;
+use sp_std::{cmp::Ordering, vec::Vec};
```

Comment:
```suggestion
use sp_std::{cmp::Ordering, prelude::*};
```

Reply:
I strongly prefer explicit imports over globs, particularly when it's a single item.

Reply:
Sure